src/domains/job/recruitment/detail/page/RecruitmentDetailPage.jsx

```
@@ -1,16 +1,15 @@
-import { useParams } from 'react-router-dom'
+import { useParams, useNavigate } from 'react-router-dom'
 import useRecruitmentPostDetail from '../hook/useRecruitmentPostDetail'
 import Spinner from '../../../../../components/web/Spinner'
-import unscrapIcon from '../../../../../assets/icons/common/common_bookmark_gray.svg'
-import scrapIcon from '../../../../../assets/icons/common/common_bookmark_pink.svg' //아직 스크랩 적용 안 했음
+import unScrapIcon from '../../../../../assets/icons/common/common_bookmark_gray.svg'
+import ScrapIcon from '../../../../../assets/icons/common/common_bookmark_pink.svg' //아직 스크랩 적용 안 했음
```

ESLint

[error] 'ScrapIcon' is defined but never used.

(no-unused-vars)